src/Microsoft.ServiceFabric.Actors/ActorId.cs

```diff
@@ -5,12 +5,15 @@
 
 namespace Microsoft.ServiceFabric.Actors
 {
+    extern alias Microsoft_ServiceFabric_Internal;
+
     using System;
-    using System.Fabric.Common;
     using System.Globalization;
     using System.Runtime.Serialization;
     using System.Text;
     using Microsoft.ServiceFabric.Services;
+    using ReleaseAssert = Microsoft_ServiceFabric_Internal::System.Fabric.Common.ReleaseAssert;

```

Comment:
Should we add a file like _alias.cs which has all the alias listed under some common namespace and then include `Microsoft.ServiceFabric.Services.Common` in other files instead of System.Fabric.Common ?

https://msazure.visualstudio.com/One/_git/WindowsFabric?path=%2Fsrc%2Fprod%2Fsrc%2Fmanaged%2FFabricUS%2Fexe%2F_aliases.cs&_a=contents&version=GBdevelop

Reply:
It didnt work for me somehow. Even in fabricUS project I have seen instances where they used extern inside the non alias cs file. I will leave it like this for now.